c2corg_ui/static/partials/trackdownload.html

```diff
@@ -0,0 +1,2 @@
+<button class="btn btn-primary" ng-click="trackCtrl.downloadGpx()">GPX</button>
```

Comment:
Actually I will remove the extra ``@export`` in another PR because there are a bunch of other controllers that also have it.